man/systemd-ssh-proxy.xml

```diff
@@ -24,7 +24,7 @@
 
   <refsynopsisdiv>
     <programlisting>
-Host unix/* vsock/*
+Host unix/* vsock/* vsock-indirect/*
```

Comment:
that's quite hard to type. maybe use a shorter prefix?

maybe:

* `vsocki/…`
* `ivsock/…`
* `vsock-ep/…` (for "entry point")

or so?

Reply:
or vsockmux/*?

vsock-mux/*?

Reply:
`vsock-mux/` seems fine to me